components/button.css

```diff
@@ -0,0 +1,201 @@
+.btn{
+    line-height: 1.5;
+    border: 1px solid transparent;
+    cursor: pointer;
+    padding: 6px 10px;
```

Comment:
You should try using rem

Reply:
I used px for measures less than or equal to 10px, as was suggested in the session. Still will think about this